iree/turbine/kernel/wave/templates/extend_attention.py

```diff
@@ -158,7 +158,7 @@ def extend_attention(
             N_KV, H, D_Q, ADDRESS_SPACE, wave_input_dtype, k_cache_layout
         ],
         v_cache: tkl.Memory[
-            H, D_KV, N_KV, GLOBAL_ADDRESS_SPACE, wave_input_dtype, v_cache_layout
+            H, D_KV, N_KV, ADDRESS_SPACE, wave_input_dtype, v_cache_layout
```

Comment:
Does changing the address space matter for numerics or is it for performance?

Reply:
I was just making it consistent with what we do for the k_cache. It works for both address spaces. I have not measured performance yet.

Reply:
sgsg!